pom.xml

```diff
@@ -1267,11 +1194,9 @@
                 </pluginExecution>
                 <pluginExecution>
                 	<pluginExecutionFilter>
-                		<groupId>au.com.alderaan</groupId>
-                		<artifactId>
-                			eclipselink-staticweave-maven-plugin
-                		</artifactId>
-                		<versionRange>[1.0.4,)</versionRange>
+                      <groupId>de.empulse.eclipselink</groupId>
+                      <artifactId>staticweave-maven-plugin</artifactId>
```

Comment:
This plugin was changed to a different version which supports newer versions of Eclipselink.